test/jdk/java/nio/channels/DatagramChannel/ManySourcesAndTargets.java

```diff
@@ -25,7 +25,7 @@
  * @bug 8234805 8235193
  * @summary Test DatagramChannel send/receive and that receive returns the expected
  *     sender address
- * @run main ManySourcesAndTargets
+ * @run main/othervm ManySourcesAndTargets
```

Comment:
This change looks okay. Do you mind change L143 to use sender.send rather than reader.send while you are there? That will avoid at least some questions while trying to track down the underlying issue and will ensure that the test is printing out the actual sender address.

Reply:
yes, I can do that ... the suggested change is currently part of a set of changes for JDK-8264385, but no problem to add it here now for this change set